src/pycliarr/api/sonarr.py

```diff
@@ -336,4 +335,23 @@ def get_queue(
         """
         return self._get_queue(
             page, sort_key, page_size, sort_dir, options={"includeUnknownSeriesItems": include_unknown}
+
+
+    def rename_files(
+        self, file_ids: List[int], serie_id: int
+    ) -> Dict[str, Any] | List[Dict[str, Any]]:
```

Comment:
this is not valid python typing. you should use union. i've created the `json_data` type for this.

Reply:
fixed with your patch, thanks!